requirements.in

```diff
@@ -26,7 +26,7 @@ https://github.com/fruch/repodataParser/archive/py3.zip ; python_version > '3'
 pytest==4.6.4
 tcconfig==0.26.0
 selenium==3.141.0
-mysql-connector-python==8.0.19
+mysql-connector-python==8.0.20
```

Comment:
this was change cause of a conflict in packages.
FYI our code that uses it doesn't really work anyhow, see #3069